test/utils/P256.t.sol

```diff
@@ -110,9 +112,18 @@ contract P256Test is Test {
             uint256 y = uint256(vector.readBytes32(".y"));
             bytes32 hash = vector.readBytes32(".hash");
 
-            if (uint256(s) <= _MALLEABILITY_THRESHOLD) {
+            if (s <= _N) {
                 assertEq(
-                    P256.verify_sig(hash, r, s, x, y),
+                    P256.verify_sig(
+                        hash,
+                        r,
+                        /**
+                         * @dev Flip the `s` parameter if it is higher than the malleability threshold.
+                         */
+                        (s > _MALLEABILITY_THRESHOLD) ? (_N - s) : s,
```

Comment:
FYI there's a case (#1) that tests for signature malleability -- I imagine that by flipping the s param for all, it cancels that test

Reply:
I have a specific malleability test [here](https://github.com/pcaversaccio/snekmate/blob/main/test/utils/P256.t.sol#L125-L129). So I think this should be fine.